tests/CMakeLists.txt

```
@@ -47,6 +47,13 @@ function(add_one_test)
             -Werror
     )
 
+    # Include directories.
+    target_include_directories(
```

Add a note to explain why we do this.

```suggestion
    # Tests must include any helper file under the 'tests' directory with the full relative path.
    target_include_directories(
```

---

OK! Clearer indeed.